zkvm/spec/ZkVM.md

```diff
@@ -204,11 +204,11 @@ bound to its [Pedersen commitment](#pedersen-commitment).
 
 A [point](#point) that represents a commitment to a secret scalar can be turned into a variable using the [`var`](#var) instruction.
 
-A cleartext [scalar](#scalar) can be turned into a single-term [expression](#expression-type) using the [`const`](#const) instruction (which does not allocate a variable). Since we do not need to hide their values, a Variable is not needed.
+A cleartext [scalar](#scalar) can be turned into a single-term [expression](#expression-type) using the [`const`](#const) instruction (which does not allocate a variable). Since we do not need to hide their values, a Variable is not needed to represent the cleartext constant.
 
 Variables can be copied and dropped at will, but cannot be ported across transactions via [outputs](#output-structure).
 
-Examples of variables: [value quantities](#value-type) and [time bounds](#time-bounds).
+[Value quantities and flavors](#value-type) are represented as variables.
```

Comment:
question: In the definition of `Value type` we have "Both quantity and flavor are represented as variables." Is this fine?

Reply:
Do you mean it's redundant to mention this here? I just wanted to show an example of variables.